Cython/Shadow.pyi

```diff
@@ -100,3 +103,19 @@ class typedef(CythonType, Generic[_T]):
 #    def __init__(self) -> None: ...
 
 #def fused_type(*args: Tuple[_T]) -> Type[FusedType[_T]]: ...
+
+class _EmptyDecoratorAndManager(object):
+    def __call__(self, x: _T) -> _T: ...
+
+    def __enter__(self) -> None: ...
+
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc: Optional[BaseException],
+        tb: Optional[TracebackType]
+    ) -> None: ...
+
+cclass = ccall = cfunc = _EmptyDecoratorAndManager()
+
+def declare(t: Optional[Callable[..., _T]] = None, value: Any = ..., **kwds: Any) -> Optional[_T]: ...
```

Comment:
I don't follow, that argument defaults to None.

Reply:
We do not want to update pyi with every small change in py, especially since stubs are intended for information about types, not about implementation, using `...` we show that there must be a default value, which can be found in py.

Reply:
you can look [here](https://mypy.readthedocs.io/en/stable/stubs.html#stub-file-syntax)

Reply:
@Bluenix2 are you going to include the suggested changes?

Reply:
this is resolved, isn't it?

Reply:
Ah, yes. `t` is still Optional, but `= None` has been changed to `= ...`